extension-manifest-v3/src/utils/opera-serp.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import { store } from 'hybrids';
+
+import Options from '/store/options.js';
+
+const NOTIFICATION_SHOW_LIMIT = 4;
+const NOTIFICATION_DELAY = 7 * 24 * 60 * 60 * 1000; // 7 days in milliseconds
+const TEST_COOKIE_NAME = 'ghostery:opera:cookie:test';
```

Comment:
just to be on a safe side - how about we make this name random to prevent detection?

Reply:
During the test, the cookie is immediately removed after successful creation, so I doubt that any other code can detect it (and it sets `httpOnly` - so no other JS can access it).